### PR TITLE
[Intel oneDNN] static analysis scan fixes based on r2.6 release tag

### PR DESCRIPTION
**Pull request by nammbash**
Few fixes based on r2.6 release tag to fix some static code analyzer complains

### CODE REVIEW

**Comment by gbaned** [comment]
@penpornk  Can you please review this PR ? Thanks!

File: tensorflow/core/graph/mkl_graph_util.h [hunk 1]
```diff
@@ -299,7 +299,9 @@ static inline bool IsMklOp(const string& op_name, DataType T) {
 
 static inline bool IsMklOp(const Node* n) {
   DataType T;
-  return GetNodeAttr(n->def(), "T", &T).ok() && IsMklOp(n->type_string(), T);
+  bool isOpMkl = false;
+  isOpMkl = GetNodeAttr(n->def(), "T", &T).ok() && IsMklOp(n->type_string(), T);
+  return isOpMkl;
```
Comment (penpornk) [inline tensorflow/core/graph/mkl_graph_util.h:304]: What do we accomplish by storing the results in a bool variable first?

**Review by penpornk — COMMENTED** [review]
Thank you for the PR! Could you please post the error message?

**Comment by gbaned** [comment]
@nammbash Can you please check @penpornk's comments and keep us posted ? Thanks!

File: tensorflow/core/graph/mkl_graph_util.h (hunk 1 above)
Comment (nammbash) [reply tensorflow/core/graph/mkl_graph_util.h:304]: Thank you. This was an interesting why Static Analysis is complaining!  The message below is copied below for reference.

_"Non-void function does not return value
/jenkins/workspace/klocwork_scan/repo-to-scan/tensorflow/core/graph/mkl_graph_util.h:284 | IsMklOp()"_

Comment (nammbash) [reply tensorflow/core/graph/mkl_graph_util.h:304]: image

**Comment by gbaned** [comment]
@penpornk Can you please assist on above comments from @nammbash. Thanks!

**Comment by gbaned** [comment]
@penpornk Any update on this PR? Please. Thanks!

File: tensorflow/core/util/mkl_util.h [hunk 2]
```diff
@@ -1838,8 +1838,7 @@ class MklPrimitiveFactory {
   static inline bool IsPrimitiveMemOptEnabled() {
     static const bool is_primitive_mem_opt_enabled = [] {
       bool value = true;
-      TF_CHECK_OK(
-          ReadBoolFromEnvVar("TF_MKL_OPTIMIZE_PRIMITIVE_MEMUSE", true, &value));
+      ReadBoolFromEnvVar("TF_MKL_OPTIMIZE_PRIMITIVE_MEMUSE", true, &value);
```
Comment (mihaimaruseac) [inline tensorflow/core/util/mkl_util.h:1841]: Why do we remove the CHECK?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gzmkl** [comment]
This PR is outdated. Please close it.

**Pull request closed without merging**